packages/libretro/reicast/package.mk

```diff
@@ -34,6 +34,15 @@ PKG_LONGDESC="Reicast is a multiplatform Sega Dreamcast emulator"
 PKG_IS_ADDON="no"
 PKG_AUTORECONF="no"
 
+pre_configure_target() {
+  CFLAGS="${CFLAGS//-flto/}"
+  CFLAGS="${CFLAGS//-ffat-lto-objects/}"
+  CXXFLAGS="${CXXFLAGS//-flto/}"
+  CXXFLAGS="${CXXFLAGS//-ffat-lto-objects/}"
+  LDFLAGS="${LDFLAGS//-flto/}"
+  LDFLAGS="${LDFLAGS//-ffat-lto-objects/}"
+}
```

Comment:
my build was failing on missing gnu/stub-32.h so I suggest adding:
```
if [ "$TARGET_ARCH" == "x86_64" ]; then
  CFLAGS="$CFLAGS -D__x86_64__ -D__LP64__
  CXXFLAGS="$CXXFLAGS -D__x86_64__ -D__LP64__"
fi
```

Reply:
you can disregard this comment. this is a mistake in `config/options`. I was building with simple `make image`. when no PROJECT is set, PROJECT=Generic, when no ARCH is set, TARGET_ARCH=x86_64, but ARCH is left empty. starting the build with `ARCH=x86_64 make image` works.